tests/integration-tests/utils.py

```diff
@@ -80,7 +80,16 @@ def retry_if_subprocess_error(exception):
     return isinstance(exception, subprocess.CalledProcessError)
 
 
-def run_command(command, capture_output=True, log_error=True, env=None, timeout=None, raise_on_error=True, shell=False):
+def run_command(
+    command,
+    capture_output=True,
+    log_error=True,
+    env=None,
+    timeout=None,
+    raise_on_error=True,
+    shell=False,
+    custom_cli_credentials=None,
```

Comment:
It doesn't look like this argument is used?

Reply:
Run_command() definition didnt originally had variable arguments, so the kwargs that its being sent from run_pcluster_command now has custom_cli_credentials as one of the arguments. So either I mention the variable I want to send or just add **kwargs in the run_command definition to avoid an error. I preferred adding my own custom_cli_credentials in way of avoiding sending unnecessary variables through kwargs.

Reply:
Will change this after part of our discussion